doc/tasks/phpmd.md

```diff
@@ -19,6 +19,7 @@ parameters:
         phpmd:
             whitelist_patterns: []
             exclude: []
+            report_format: text
```

Comment:
Is it possible to set this option the null and let it fall back to the defaults inside the CLI tool?

Reply:
the report format is required by phpmd and previously the task here has set it to text. 
As seen here https://github.com/phpro/grumphp/pull/766/files#diff-a5512ab2de18407a205f9c4a15f17427L67
```
        $arguments->add('text');	
```
I'm not sure what you are asking of me.

Reply:
ok, thanks for the feedback. That seems like the best choice then!